test/test_utils/__init__.py

```diff
@@ -372,46 +372,62 @@ def get_dlc_images():
 
 
 def parse_canary_images(framework, region):
-    tf1 = "1.15"
-    tf2 = "2.2"
-    mx = "1.6"
-    pt = "1.5"
+    """
+    Return which canary images to run canary tests on for a given framework and AWS region
 
+    :param framework: ML framework (mxnet, tensorflow, pytorch)
+    :param region: AWS region
+    :return: dlc_images string (space separated string of image URIs)
+    """
     if framework == "tensorflow":
         framework = "tensorflow2" if "tensorflow2" in os.getenv("CODEBUILD_BUILD_ID") else "tensorflow1"
 
+    versions = {
+        "tensorflow1": ["1.15", "1.14", "1.13"],
+        "tensorflow2": ["2.2", "2.1", "2.0"],
+        "mxnet": ["1.6", "1.4"],
+        "pytorch": ["1.5", "1.4", "1.3"]
+    }
+
     registry = PUBLIC_DLC_REGISTRY
+    framework_versions = versions[framework]
+    dlc_images = ""
+    for version in framework_versions:
+        images = {
+            "tensorflow1":
+                f"{registry}.dkr.ecr.{region}.amazonaws.com/tensorflow-training:{version}-gpu-py3 "
+                f"{registry}.dkr.ecr.{region}.amazonaws.com/tensorflow-training:{version}-cpu-py3 "
+                f"{registry}.dkr.ecr.{region}.amazonaws.com/tensorflow-training:{version}-cpu-py2 "
```

Comment:
We have all python version in TF1 - py2, py36 and py37, Do we need any additional one for them.

Reply:
We are just running on SM tags, so we will not be running on images not supported by SageMaker